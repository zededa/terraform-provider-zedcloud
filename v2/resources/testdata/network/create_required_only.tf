resource "zedcloud_network" "required_only" {
		# computed
    # id =

		# required
    name = "zedcloud_network.required_only.name"
		title = "zedcloud_network.required_only.title"
		project_id = "4754cd0f-82d7-4e06-a68f-ff9e23e75ccf"
    ip {
        dhcp = "NETWORK_DHCP_TYPE_STATIC"
        dhcp_range {
            start = "172.25.24.1"
            end = "172.25.24.3"
        }
        dns = [
            "172.25.24.254",
            "172.25.24.255"
        ]
        domain = "example.com"
        gateway = "172.25.24.254"
        mask = "255.255.0.0"
        ntp = ""
        subnet = "172.25.24.0/22"
    }
    kind = "NETWORK_KIND_V4"
}
