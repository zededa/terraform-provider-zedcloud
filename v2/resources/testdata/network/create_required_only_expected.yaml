dnslist: []
enterprisedefault: false
ip:
    deprecateddhcp: false
    dhcp: NETWORK_DHCP_TYPE_STATIC
    dhcprange:
        end: 172.25.24.3
        start: 172.25.24.1
    dns:
        - 172.25.24.254
        - 172.25.24.255
    domain: example.com
    gateway: 172.25.24.254
    mask: 255.255.0.0
    ntp: ""
    subnet: 172.25.24.0/22
kind: NETWORK_KIND_V4
name: zedcloud_network.required_only.name
projectid: 4754cd0f-82d7-4e06-a68f-ff9e23e75ccf
proxy:
    networkproxy: false
    networkproxycerts: []
    proxies: []
title: zedcloud_network.required_only.title
wireless:
    cellularcfg: {}
    type: NETWORK_WIRELESS_TYPE_UNSPECIFIED
